scripts/check_doc_ipfs_hashes.py

```diff
@@ -271,10 +271,10 @@ def check_ipfs_hashes(  # pylint: disable=too-many-locals,too-many-statements
             hash_mismatches = True
 
             if fix:
-                new_content = content.replace(doc_full_cmd, new_command)
+                content = content.replace(doc_full_cmd, new_command)
 
                 with open(str(md_file), "w", encoding="utf-8") as qs_file:
-                    qs_file.write(new_content)
+                    qs_file.write(content)
```

Comment:
This was a bug: if several hashes were mismatching in the same file, only the first one would be corrected.